src/main/routes/enter-claim-number.ts

```diff
@@ -5,14 +5,26 @@ import { ErrorHandling } from 'common/errorHandling'
 import { Paths } from 'app/paths'
 import { FormValidator } from 'forms/validation/formValidator'
 import { ClaimReference } from 'forms/models/claimReference'
-import { isNonCMCReference } from 'common/utils/isNonCMCReference'
-import { Form } from 'forms/form'
+import { isCMCReference } from 'common/utils/isCMCReference'
+import { isCCBCCaseReference } from 'common/utils/isCCBCCaseReference'
+import { Form, FormValidationError } from 'forms/form'
+import { ValidationError } from 'class-validator'
 
 function renderView (form: Form<ClaimReference>, res: express.Response): void {
   res.render(Paths.enterClaimNumberPage.associatedView, { form: form })
 }
 
 const mcolUrl = config.get<string>('mcol.url')
+const ERROR_MESSAGE: string = "Reference number is invalid. Please click on 'don't have claim reference number'"
```

Comment:
1. that sentence doesn't make sense
2. the text says `Don’t have a claim number?`
where did you get this content?